diff --git a/node_modules/oxc-parser/index.js b/node_modules/oxc-parser/index.js
index 4b1b9d0..abbc069 100644
--- a/node_modules/oxc-parser/index.js
+++ b/node_modules/oxc-parser/index.js
@@ -37,7 +37,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.android-arm64.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-android-arm64')
+            nativeBinding = require('@oxc-parser/binding-android-arm64/parser.android-arm64.node')
           }
         } catch (e) {
           loadError = e
@@ -49,7 +49,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.android-arm-eabi.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-android-arm-eabi')
+            nativeBinding = require('@oxc-parser/binding-android-arm-eabi/parser.android-arm-eabi.node')
           }
         } catch (e) {
           loadError = e
@@ -69,7 +69,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.win32-x64-msvc.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-win32-x64-msvc')
+            nativeBinding = require('@oxc-parser/binding-win32-x64-msvc/parser.win32-x64-msvc.node')
           }
         } catch (e) {
           loadError = e
@@ -83,7 +83,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.win32-ia32-msvc.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-win32-ia32-msvc')
+            nativeBinding = require('@oxc-parser/binding-win32-ia32-msvc/parser.win32-ia32-msvc.node')
           }
         } catch (e) {
           loadError = e
@@ -97,7 +97,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.win32-arm64-msvc.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-win32-arm64-msvc')
+            nativeBinding = require('@oxc-parser/binding-win32-arm64-msvc/parser.win32-arm64-msvc.node')
           }
         } catch (e) {
           loadError = e
@@ -113,7 +113,7 @@ switch (platform) {
       if (localFileExisted) {
         nativeBinding = require('./parser.darwin-universal.node')
       } else {
-        nativeBinding = require('@oxc-parser/binding-darwin-universal')
+        nativeBinding = require('@oxc-parser/binding-darwin-universal/parser.darwin-universal.node')
       }
       break
     } catch {}
@@ -124,7 +124,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.darwin-x64.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-darwin-x64')
+            nativeBinding = require('@oxc-parser/binding-darwin-x64/parser.darwin-x64.node')
           }
         } catch (e) {
           loadError = e
@@ -138,7 +138,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.darwin-arm64.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-darwin-arm64')
+            nativeBinding = require('@oxc-parser/binding-darwin-arm64/parser.darwin-arm64.node')
           }
         } catch (e) {
           loadError = e
@@ -157,7 +157,7 @@ switch (platform) {
       if (localFileExisted) {
         nativeBinding = require('./parser.freebsd-x64.node')
       } else {
-        nativeBinding = require('@oxc-parser/binding-freebsd-x64')
+        nativeBinding = require('@oxc-parser/binding-freebsd-x64/parser.freebsd-x64.node')
       }
     } catch (e) {
       loadError = e
@@ -174,7 +174,7 @@ switch (platform) {
             if (localFileExisted) {
               nativeBinding = require('./parser.linux-x64-musl.node')
             } else {
-              nativeBinding = require('@oxc-parser/binding-linux-x64-musl')
+              nativeBinding = require('@oxc-parser/binding-linux-x64-musl/parser.linux-x64-musl.node')
             }
           } catch (e) {
             loadError = e
@@ -187,7 +187,7 @@ switch (platform) {
             if (localFileExisted) {
               nativeBinding = require('./parser.linux-x64-gnu.node')
             } else {
-              nativeBinding = require('@oxc-parser/binding-linux-x64-gnu')
+              nativeBinding = require('@oxc-parser/binding-linux-x64-gnu/parser.linux-x64-gnu.node')
             }
           } catch (e) {
             loadError = e
@@ -203,7 +203,7 @@ switch (platform) {
             if (localFileExisted) {
               nativeBinding = require('./parser.linux-arm64-musl.node')
             } else {
-              nativeBinding = require('@oxc-parser/binding-linux-arm64-musl')
+              nativeBinding = require('@oxc-parser/binding-linux-arm64-musl/parser.linux-arm64-musl.node')
             }
           } catch (e) {
             loadError = e
@@ -216,7 +216,7 @@ switch (platform) {
             if (localFileExisted) {
               nativeBinding = require('./parser.linux-arm64-gnu.node')
             } else {
-              nativeBinding = require('@oxc-parser/binding-linux-arm64-gnu')
+              nativeBinding = require('@oxc-parser/binding-linux-arm64-gnu/parser.linux-arm64-gnu.node')
             }
           } catch (e) {
             loadError = e
@@ -231,7 +231,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.linux-arm-gnueabihf.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-linux-arm-gnueabihf')
+            nativeBinding = require('@oxc-parser/binding-linux-arm-gnueabihf/parser.linux-arm-gnueabihf.node')
           }
         } catch (e) {
           loadError = e
@@ -246,7 +246,7 @@ switch (platform) {
             if (localFileExisted) {
               nativeBinding = require('./parser.linux-riscv64-musl.node')
             } else {
-              nativeBinding = require('@oxc-parser/binding-linux-riscv64-musl')
+              nativeBinding = require('@oxc-parser/binding-linux-riscv64-musl/parser.linux-riscv64-musl.node')
             }
           } catch (e) {
             loadError = e
@@ -259,7 +259,7 @@ switch (platform) {
             if (localFileExisted) {
               nativeBinding = require('./parser.linux-riscv64-gnu.node')
             } else {
-              nativeBinding = require('@oxc-parser/binding-linux-riscv64-gnu')
+              nativeBinding = require('@oxc-parser/binding-linux-riscv64-gnu/parser.linux-riscv64-gnu.node')
             }
           } catch (e) {
             loadError = e
@@ -274,7 +274,7 @@ switch (platform) {
           if (localFileExisted) {
             nativeBinding = require('./parser.linux-s390x-gnu.node')
           } else {
-            nativeBinding = require('@oxc-parser/binding-linux-s390x-gnu')
+            nativeBinding = require('@oxc-parser/binding-linux-s390x-gnu/parser.linux-s390x-gnu.node')
           }
         } catch (e) {
           loadError = e
